straxen/url_config.py

```diff
@@ -232,6 +232,13 @@ def format_arg(arg: str, **kwargs):
     return arg.format(**kwargs)
 
 
-@URLConfig.register('interpolatingmap')
+@URLConfig.register('itp_map')
 def load_map(some_map, method='WeightedNearestNeighbors', **kwargs):
+    '''Make an InterpolatingMap'''
     return straxen.InterpolatingMap(some_map, method=method)
+
+@URLConfig.register('bodega')
+def load_value(name: str, version='v0'):
+    '''Load a number from BODEGA file'''
+    nT_numbers = straxen.get_resource("XENONnT_numbers.json", fmt="json")
+    return nT_numbers[name][version]["value"]
```

Comment:
making a PR now. In any case a more general comment is that we should limit the reuse of keyword arguments as much as possible, so here i would go with `bodega_version` or something similar since version is already use quite ofter with cmt.

Reply:
@ershockley sure, if we are starting it for a lot of things, it's better to add a elegant alias as you did here

I'm just trying to think ahead and prevent adding too many (unused) protocols as removing them later in an attempt to simplify is never going to happen especially if it's easy to use nice features that Yossi keeps adding: https://github.com/XENONnT/straxen/pull/826

We can always add it later if needed

Reply:
I renamed the bodega_version for now since that's a minimum needed change. Did you also want to use #826 instead of adding this protocol? I am on the fence. On one hand its completely possible to use nested access but on the other hand it reduces the readability of the config URL.
In general my approach would be that we should aim to have enough protocols that make the default configs simple and readable, and use the chaining feature for flexibility when you want to change something without releasing a new straxen version.
But on the third hand, having a million redundant protocols will make a mess for maintenance...

Reply:
also fine, let's wrap now then. I'll just take my loss again

Reply:
I vote for keeping the bodega protocol. It's very simple and clear what it is doing.